app/javascript/styles/mastodon/forms.scss

```
@@ -722,7 +722,7 @@ code {
     }
   }
 
-  @media screen and (max-width: 740px) and (min-width: 441px) {
+  @media screen and (441px <= width <= 740px) {
```

```suggestion
  @media screen and (440px < width <= 740px) {
```

I think the 1px is unnecessary by not being `min-width`.